nixos/modules/system/boot/stage-2-init.sh

```diff
@@ -91,6 +91,12 @@ if ! mountpoint -q /dev/shm; then
 fi
 mkdir -m 0755 -p /dev/pts
 [ -e /proc/bus/usb ] && mount -t usbfs usbfs /proc/bus/usb # UML doesn't have USB by default
+
+# Clean tmp directory, if requested
+if [ -n "@cleanTmpDir@" ]; then
+    rm -rf /tmp
```

Comment:
Are are you sure about that command? This will remove the entire mount point, which doesn't seem right to me.